bin/MadGraph5_aMCatNLO/cards/double_prod_T1_3body_decay_MCHM5_C9/double_prod_T1_3body_decay_MCHM5_C9_run_card.dat

```diff
@@ -86,7 +86,13 @@ $DEFAULT_PDF_MEMBERS = reweight_PDF      ! if pdlabel=lhapdf, this is the lhapdf
 #**********************************************************
   .false.	=  auto_ptj_mjj   ! Automatic setting of ptj and mjj
 #**********************************************************
-#                                                                    
+#
+#*********************************************************************
+# Phase-Space Optim (advanced)
+#*********************************************************************
+   1 =  hard_survey ! force to have better estimate of the integral at survey for difficult mode like interference          
+#*********************************************************************
+#                                                               
```

Comment:
This option doesn't exist in 2.6.5. Where did you find it?

Reply:
I use a more recent version, 2.9.6 in which this parameter is used to overcome issues MadGraph encounters when generating the events for processes with difficult mode like interference. This happens in the processes: "double_prod_T1_3body_decay_MCHM5_C9" and "double_prod_X53_3body_decay_MCHM5_C9". Would it be possible to use this more recent 2.9.6 version at least for these two cases?

Reply:
I'm afraid that master branch is still set to 2.6.5 and the plan is to update it to 3.3.0 for Run III and the patches are currently being tested. @agrohsje should we include this as a pLHE request then?

Reply:
As replied to Sapta by email two days ago, there is no problem for us provide the LHE files for the 3 resonant processes on double production of T1 and X53 including or not 3 body decay within the MCHM5_C9 scenario. We thus wait for the answer from @agrohsje.